pkg/cmd/server/start.go

```diff
@@ -336,6 +338,9 @@ func start(cfg *config, args []string) error {
 			osmaster.RunAPI(auth, osmaster, &origin.SwaggerAPI{})
 		}
 
+		// TODO: recording should occur in individual components
+		record.StartRecording(osmaster.KubeClient().Events(""), kapi.EventSource{Component: "master"})
```

Comment:
I'm doing what the scheduler does - @derekwaynecarr what namespace *should* we send when we want to record events from infrastructure components?  We need to fix this upstream too.